frontend/src/store/userLibraries.ts

```diff
@@ -141,7 +141,8 @@ class UserLibrariesStore {
         state.value.homeSections.audioResumes = audioResumes;
       }
     } catch (error) {
-      useSnackbar(error as string, 'error');
+      // TODO: actual user-facing error messages
```

Comment:
This seems to be the only place in the codebase we try to snackbar an error. I'm happy to remove these in favor of a `console.error` or remove the TODOs if a real translated error isn't ever going to be a thing.

Reply:
I think snackbars are better than silently throwing errors for users that could be meaningful. So yeah we could add some proper errors :)

Reply:
Follow up question: Should we SnackBar the raw JS error like in this case, or should we show the generic error message in the translation file (`errors.anErrorHappened`) and console.error the actual JS error.

Reply:
I'd do your second option: snackbar a generic message and console.error the real one :)

Reply:
Yeah, all our snackbars needs a lot of cleanup (but probably that it's more worth it for another PR, I have this pending on my mind but need to write it to the roadmap at some point).

We have a lot of messages for a lot of actions that can fail (e.g like an item, mark as watched, etc...), where all of these could be resumed to the following messages:

* An error has happened
* The operation couldn't be performed.